package.json

```diff
@@ -8,7 +8,7 @@
     "build-dev": "webpack --mode=development",
     "watch": "webpack --mode=development --watch",
     "gobuild": "./buildandrun.sh",
-    "lint": "npx eslint src/js/**"
+    "lint": "npx eslint \"src/js/**/*.{js,jsx}\""
```

Comment:
We should also lint the .ts files

Reply:
Yes but I want to wait till I convert everything to typescript just cause, there are like a couple situations where there are unused exports/variables that would be picked up by the linter but we need the full typescript conversion for them to go away. Otherwise it is not that bad cause my IDE also tries to lint stuff and I have prettier.